src/Workspaces/Core/Portable/Utilities/ImmutableArrayExtensions.cs

```diff
@@ -11,7 +13,7 @@ internal static class ImmutableArrayExtensions
         internal static bool Contains<T>(this ImmutableArray<T> items, T item, IEqualityComparer<T> equalityComparer)
```

Comment:
The comparer is optional right?

```suggestion
        internal static bool Contains<T>(this ImmutableArray<T> items, T item, IEqualityComparer<T>? equalityComparer)
```

Reply:
It is, and you caught that in your PR. Not sure if it's worth fixing mine too or just let you merge yours.

Reply:
Going to let you take this one rather than churn this further.